contribs/pom.xml

```diff
@@ -249,6 +249,32 @@
 		</dependency>
 	</dependencies>
 
+	<!-- Some commonly used dependencies with fixed versions, please try to avoid replicating the versions in contrib poms -->
+	<dependencyManagement>
```

Comment:
Should we maybe have this section I level up so that everything is kept in sync?

Reply:
One more thought: Maybe we should also do that for other dependencies used in many places (esp. geotools)?

Reply:
It is not possible to put it higher up, because there is no common parent pom for contribs, core, etc. But maybe that is something that should be changed...

Geotools is only imported directly by the core, so putting it there won't have any effect. I think one needs to define a BOM that can be imported by other projects, in order to fixate such dependencies that might cause problems.

Reply:
Here's the parent pom: https://github.com/matsim-org/matsim-libs/blob/master/pom.xml.

Some geotools dependencies are imported by specific contribs:
https://github.com/matsim-org/matsim-libs/blob/f2ef30cd8b688737e0c71e636e3dc69d5f3b7918/contribs/accessibility/pom.xml#L92-L94
But I agree that without BOM it will be too much of manual work.

Reply:
The pom in contrib does not declare any parent, so I think changes in the mentioned file does not affect it, but I need to double-check.

Reply:
Oh, you're right. I am so much used to always having parents in mvn submodules, but not the case here...